Makefile

```diff
@@ -334,28 +322,21 @@ ifeq ($(BUILD_ENTERPRISE_READY),true)
 	rm -f config/*.key
 endif
 
-test-server: test-te test-ee
+test: test-te test-ee
```

Comment:
Jenkins needs the test-server target. Please add that back as an alias.

Reply:
Wouldn't be too hard to just have Jenkins try both targets, would it? I think "test" is preferable now that the server is the only thing in the repo.

Reply:
Ya. That's way better, let me do that.